internal/pkg/vm/testing.go

```diff
@@ -278,6 +278,15 @@ func (ts *testStorage) Get(cid cid.Cid, obj interface{}) bool {
 	return true
 }
 
+func (ts *testStorage) GetRaw(cid cid.Cid) ([]byte, bool) {
+	node, err := cbor.WrapObject(ts.state, types.DefaultHashFunction, -1)
```

Comment:
No matter what we do here it will need more attention so nbd.  But you could help me out a bit by grabbing bytes from encoding.Encode and then passing them to [`cbor.Decode`](https://github.com/ipfs/go-ipld-cbor/blob/master/node.go#L95) to make an ipld node directly around bytes.

Reply:
Let's talk offline